app/presenters/workshop_presenter.rb

```diff
@@ -77,13 +77,26 @@ def attending_organisers
   end
 
   def attendee_array
```

Comment:
The original attendee_array function was complicated. I wouldn't be confident refactoring it until I added more tests to see what it's up to. As it stands to add a column to the CSV you have to change the array of items in 3 places! Two directly in attended_array and one in a called sub-method attending_organisers.

```
  def attending_organisers
    organisers.map do |o|
      [o.full_name, 'ORGANISER']
    end
  end

```
Clearly, this isn't a user friendly function and the lack of dryness means if someone "just" wants to add another column there's a good chance of mistakes.